internal/test/wmcb/deploy/deploy.yaml

```diff
@@ -0,0 +1,39 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: windows-machine-config-bootstrapper
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      name: windows-machine-config-bootstrapper
+  template:
+    metadata:
+      labels:
+        name: windows-machine-config-bootstrapper
+    spec:
+      hostNetwork: true
+      volumes:
+      - name: cloud-private-key
+        secret:
+          secretName: cloud-private-key
+      - name: aws-creds
+        secret:
+          secretName: aws-creds
```

Comment:
Why do we still need the creds?

Reply:
We need aws creds to create machineSets. We are fetching the Subnet, etc